### DISCUSSION TITLE
ZEPETO BUILD IT

### DISCUSSION THREAD

**Discussion by Darkzpt** [naverz/zepeto-studio-kor#278] [5. Q&A]
![image](https://user-images.githubusercontent.com/97772423/151164670-a005331e-8822-466c-8800-6ad3b92c0d99.png)
'제페토 빌드잇'에 문제가 있나요? 로딩에 합격할 수 없습니다. 저는 아직 초보라 무슨 일이 일어나고 있는지 잘 모르겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 제페토 빌드잇 관련해서 계속 같은 문제가 발생하고 있는지 확인 부탁드립니다. 
감사합니다.
